src/mud/examples/poisson.py

```diff
@@ -126,13 +126,99 @@ def spline_objective_function_2d(lam, aff=10000):
     return np.linalg.norm(g - vals)
 
 
+def plot_qoi_ex(mud_prob, num_components: int, **kwargs):
+    fig = plt.figure(figsize=(10, 5))
+    for i in range(num_components):
+        ax = fig.add_subplot(1, 2, i + 1)
+        mud_prob.plot_params_2d(ax=ax, y=i, contours=True, colorbar=True)
+        if i == 1:
+            ax.set_ylabel("")
+    if kwargs.get("save_path"):
```

Comment:
Should be 
`if kwargs,get("save_path") is not None:`
See [PEP 8](https://peps.python.org/pep-0008/)